openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners_assessment.html

```diff
@@ -0,0 +1,58 @@
+{% load i18n %}
+{% spaceless %}
+
+    {% block body %}
+    <div class="staff__grade__form ui-toggle-visibility__content" data-submission-uuid="{{ submission.uuid }}">
+        <div class="wrapper--staff-assessment">
+            <div>
+                <p>{% trans "Give this learner a grade using the problem's rubric." %}</p>
+            </div>
+
+            <div>
+                <article class="staff-assessment">
+                    <div class="staff-assessment__display">
+                        <header class="staff-assessment__display__header">
+                            <h4 class="staff-assessment__display__title">
+                                {% blocktrans %}
+                                    Response for: {{ student_username }}
+                                {% endblocktrans %}
+                            </h4>
+                        </header>
+
+                        {% include "openassessmentblock/oa_submission_answer.html" with answer=submission.answer answer_text_label="The learner's response to the question above:" %}
+
+                        {% include "openassessmentblock/oa_uploaded_file.html" with file_upload_type=file_upload_type file_url=staff_file_url header="Associated File" class_prefix="staff-assessment" show_warning="true" %}
+                    </div>
+
+                    <form class="staff-assessment__assessment" method="post">
+                        {% include "openassessmentblock/oa_rubric.html" with rubric_type="staff" %}
+                    </form>
+                </article>
+            </div>
+
+            <div>
+                <div class="message message--inline message--error message--error-server">
+                    <h4 class="message__title">{% trans "We could not submit your assessment" %}</h4>
+                    <div class="message__content"></div>
+                </div>
+
+                <ul class="list list--actions">
+                    <li class="list--actions__item submit_assessment--action">
+                        <button type="submit" class="action action--submit is--disabled">
+                            <span class="copy">{% trans "Submit Assessment" %}</span>
```

Comment:
@catong what do you think of the wording of these buttons? Is it OK if we make the first button ("Submit Assessment") text consistent with the staff grade override button (which is currently "Submit your assessment")?